docs/changelog.md

```diff
@@ -25,6 +25,9 @@ permalink: /changelog/
 * **browser-engine-gecko(-beta/nightly)**, **concept-engine**
   * Added simplified `Media.state` derived from `Media.playbackState` events.
 
+* **lib-crash**
+  * Now supports Breadcrumbs.  Use the 'recordCrashBreadcrumb()' to record Breadcrumbs if the underlying crash reporter service supports it.
```

Comment:
nit: It's nice to have some sample code here when we add new APIs so when folks read what's new, they know how to use it.

Reply:
Agreed.  I'll add it.